packages/inscription-view/src/components/parts/common/permission/Permission.tsx

```diff
@@ -14,9 +14,10 @@ interface PermissionProps {
   config: StartPermission;
   defaultConfig: StartPermission;
   updatePermission: DataUpdater<StartPermission>;
+  updatePermissionAddRole: DataAdder<StartPermission>;
```

Comment:
I'm not happy with this solution. Maybe you have an idea. If I only would work with `updatePermission`, then the Role select from the role browser would replace the whole array of roles instead of adding one.